skyportal/tests/tools/test_sky.py

```diff
@@ -0,0 +1,71 @@
+import numpy as np
+from astropy.time import Time
+
+
+def test_airmass(public_source, ztf_camera):
+    telescope = ztf_camera.telescope
+    times = Time(np.linspace(58000, 59000), format='mjd')
+    airmass_calc = public_source.airmass(telescope, times)
+    airmass_true = np.asarray([np.inf, np.inf, 4.259917450786537,
```

Comment:
Where do these numbers come from?  Is it possible to only query the airmass at one or two points in time, and to hand-verify the result instead?

Reply:
Redid this using solid numbers from an independent calculation (http://www.briancasey.org/artifacts/astro/airmass.cgi?observatory=p&obslong=&obslat=&actual=&effective=&site=&zone=&zonename=&zoneabbrev=&daylight=1&year=2020&month=07&day=21&starname=pubsource&rightasc=00+00+00&declin=00+00+00&epoch=2000&altitude=-18&command=display)